### PR TITLE
use MD5 through JSI with react-native-quick-md5

### PR DESCRIPTION
**Pull request by osdnk**
Fixes RNBW-3325

## What changed (plus any additional context for devs)

I used the MD5 function through JSI

Methodology:image

results:
Android:
before:
LOG TOTAL 66 72602987289429

After
TOTAL 14 525776863098145

iOS:
before:
TOTAL 44 21558330953121
After:
TOTAL 5 143124938011169

## PoW (screenshots / screen recordings)

Should be no-op, images should load 
Discover

Discover

## Dev checklist for QA: what to test

Should be no-op

rel: https://github.com/rainbow-me/rainbow-scripts

### CODE REVIEW

**Review by jkadamczyk — APPROVED** [review]
LGTM, great improvement!

File: package.json [hunk 1]
```diff
@@ -214,6 +214,7 @@
     "react-native-os": "brunobar79/react-native-os#41a08e45260b8c1c53c77ad00e13ae954cb9337c",
     "react-native-palette-full": "1.2.0",
     "react-native-permissions": "3.0.5",
+    "react-native-quick-md5": "https://github.com/craftzdog/react-native-quick-md5",
```
Comment (brunobar79) [inline package.json:217]: U need to lock a specific commit so there are no surprises

**Review by brunobar79 — COMMENTED** [review]
Looks good after dep pinning

**Review by terrysahaidak — APPROVED** [review]
Looks good to me too after the version pinning!

**Review by skylarbarrera — APPROVED** [review]
👌

**Review by tchayen — APPROVED** [review]
Checked on Android and iOS. Nothing broke, looks good!

**Review by brunobar79 — APPROVED** [review]

**Pull request merged**